Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:8.0.14-apache
+FROM php:8.1-apache
```

Comment:
Can we keep the 8.0 version currently Fusio and the components are not fully tested for PHP 8.1